test_runner/batch_others/test_normal_work.py

```diff
@@ -41,6 +42,7 @@ def test_normal_work(neon_env_builder: NeonEnvBuilder):
     """
 
     env = neon_env_builder.init_start()
+    neon_env_builder.num_safekeepers = num_safekeepers
     pageserver_http = env.pageserver.http_client()
 
     for _ in range(3):
```

Comment:
I think you intended to change the hardcoded 3 to num_timelines here:

```suggestion
    for _ in range(num_timelines):
```

Reply:
yes, thanks